styles/_top-nav.scss

```diff
@@ -0,0 +1,64 @@
+.top-nav {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+  text-transform: uppercase;
+}
+
+.links {
+  display: flex;
+  flex-direction: row;
+  padding: 0 25%;
+  background: $brand-black;
+  color: $brand-white;
+  font-size: 85%;
+}
+
+.logo {
+  display: flex;
+  justify-content: center;
+  width: 50%;
```

Comment:
Since images will automatically figure out their height, you can leave this with just the height. If you need room between items, use margin.